flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -137,6 +137,11 @@ under the License.
 						<configuration>
 							<compilerArgument>-Xlint:deprecation</compilerArgument>
 							<failOnWarning>true</failOnWarning>
+							<excludes>
+								<exclude>
+									org/apache/flink/streaming/examples/iteration/IterateExample.java
```

Comment:
We should explain that this example is temporarily preserved only for testing purpose.